README.md

```diff
@@ -41,13 +41,13 @@ You can see the diff between the current DB schema and the ghsync schema here:
 +++ database/migrations/000001_init.up.sql	2019-09-30 12:27:48.783414881 +0100
```

Comment:
I think some automatic conversion maybe of tabs into spaces, or removal of spaces in empty lines and stuff like that.